files/ansible/deploy_mtt.yml

```diff
@@ -56,16 +63,23 @@
             - "{{ test_plans }}"
     when:
       - inventory_hostname in groups['sms']
-        
-  - name: Create directory because MTT
+
+  - name: Create Directory for Snapshot module template
     file:
-            path: "/opt/mtt/samples/python/"
+            path: "/opt/ohpc/pub/moduledeps/gnu8/openmpi4"
             state: directory
     when:
       - inventory_hostname in groups['sms']
+        
+  - name: Put in the Lmod OpenMPI Snapshot module template
+    template:
+            src: "templates/ompi_snapshot_lmod.j2"
+            dest: "/opt/ohpc/pub/moduledeps/gnu8/openmpi4/4.1.0"
```

Comment:
perhaps these names (gnu8, openmpi4, 4.1.0) should be variables in a config file?